SPECS/grep/grep.spec

```diff
@@ -1,31 +1,44 @@
 Summary:        Programs for searching through files
 Name:           grep
 Version:        3.1
-Release:        4%{?dist}
+Release:        5%{?dist}
 License:        GPLv3+
-URL:            http://www.gnu.org/software/grep
-Group:          Applications/File
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+Group:          Applications/File
+URL:            http://www.gnu.org/software/grep
```

Comment:
Nit: since we're moving this, can we also make it HTTPS?

Reply:
yes, fixed